BlueprintUI/Sources/Environment/Environment.swift

```diff
@@ -34,30 +34,77 @@ import UIKit
 ///             }
 ///         }
 ///     }
-public struct Environment {
+public struct Environment : Equatable {
```

Comment:
Now that this is Equatable; we should check this when setting the environment property on BlueprintView